### PR TITLE
Set permissions for the blob-driver storage directory

### PR DESCRIPTION
**Pull request by jhatcher1**
## Description
Sets blob-driver host volume permissions for replicated installs.

### CODE REVIEW

File: vmware.sh [hunk 1]
```diff
@@ -273,6 +273,10 @@ else
     logSuccess "Linked /var/lib/docker to data volume"
 fi
 
+# For local blob storage
+mkdir -p /var/lib/docker/blob-driver
+chown -R 9999:9999 /var/lib/docker/blob-driver
```
Comment (ms1111) [inline vmware.sh:278]: Makes sense.

I'm actually surprised /var/lib/docker is still the right path. I guess we probably keep some custom stuff under there that's not related to the Docker engine itself, which we no longer install.

**Review by ms1111 — APPROVED** [review]

**Comment by ms1111** [comment]
Is there a place in Confluence we can tuck away some release notes for the next upgrade? There are a few commands people are going to have to run after the AKS change and the non-root change.

File: vmware.sh (hunk 1 above)
Comment (jhatcher1) [reply vmware.sh:278]: At least for now, the on-prems still install docker. We'll probably have to change this path at some point to migrate the existing installs over to containerd.

**Comment by jhatcher1** [comment]
I've started an on-prem release notes section here: https://mindbridge-ai.atlassian.net/wiki/spaces/OPS/pages/2460418076/3.10

I'll copy it to the release in replicated.

File: vmware.sh (hunk 1 above)
Comment (ms1111) [reply vmware.sh:278]: Oh, I thought we'd migrated GT to containerd, and then OAG was containerd from the start?

Comment (jhatcher1) [reply vmware.sh:278]: I'm not sure about GT, in the kurl-installer.yaml we still have:
```
  docker:
    version: latest
```

I made some notes about how to manually replace it for an on-prem install, but afaik we haven't done it for any existing customers: https://mindbridge-ai.atlassian.net/wiki/spaces/SBE/pages/1855029496/Replacing+docker+with+containerd+for+on-prem+installs

**Review by edoyon90 — APPROVED** [review]
Once this is merged I'll take a look at prompting the user for which devices to use for /data and /backup in the vmware script

**Pull request merged**